apps-rendering/src/components/Footer/Footer.defaults.tsx

```diff
@@ -0,0 +1,66 @@
+// ----- Imports ----- //
+
+import type { SerializedStyles } from '@emotion/react';
+import { css } from '@emotion/react';
+import { background } from '@guardian/common-rendering/src/editorialPalette';
+import type { ArticleFormat } from '@guardian/libs';
+import {
+	breakpoints,
+	from,
+	neutral,
+	remSpace,
+	textSans,
+} from '@guardian/source-foundations';
+import FooterContent from 'components/FooterContent';
+import type { FC } from 'react';
+import { darkModeCss } from 'styles';
+
+// ----- Component ----- //
+
+const defaultStyles = (format: ArticleFormat): SerializedStyles => css`
+	border-width: 0 1px;
+	${textSans.small({ lineHeight: 'regular' })};
+	margin-left: 0;
+	margin-right: 0;
+	padding-left: ${remSpace[3]};
+	padding-right: ${remSpace[3]};
+	padding-top: ${remSpace[4]};
+	padding-bottom: ${remSpace[4]};
+
+	${from.wide} {
+		margin: 0 auto;
+		width: ${breakpoints.wide}px;
+	}
+
+	a {
+		${textSans.small({ lineHeight: 'regular' })};
+		color: ${neutral[7]};
+		text-decoration: underline;
+	}
+
+	${darkModeCss`
+        color: ${neutral[60]};
+		background-color: ${background.articleContentDark(format)};
```

Comment:
maybe one more level of indentation needed here?

Reply:
strange.. looks fine in this diff (and VSCode) but not in the "Files changed" tab..

Reply:
Looks like L42 and L44 are indented with spaces, and L43 is a tab

Reply:
I think prettier tends to get confused by the syntax of the `darkModeCss` helper, you often see this cropping up. It looks different in VSCode and GH because I think we generally tend to have tabs set to 4 spaces in VSCode, whereas GH's default is 8 spaces.

I've pushed a commit to change these to tabs.